### DISCUSSION TITLE
Why pySCF is not supported on Windows?

### DISCUSSION THREAD

**Discussion by bronze7** [pyscf/pyscf#2660] [Q&A] [answered]
I am new to this. Got into quantum chemistry simulation where I needed pySCF but found out that it is impossible. I first installed cmake as a perquisite but showed error for BLAS API which I couldn't find a way to install (if possible kindly mention how). I guess my solution will be use a VM for linux. If I use WSL (Windows Subsystem for Linux), install pyscf in it, run code editor from it and use other libraries (python, qiskit) from windows, will it work?

**Comment by sunchong137** [comment, marked as answer]
Hi Bronze7, thank you for your interest in using PySCF.

Yes, it is recommended to install WSL provided by the Windows App shop, and then install PySCF as you would do on a Linux system. I am actually also using PySCF on a Windows 11 system!

Hope this helps!
Chong

**Comment by bronze7** [comment]
> Hi Bronze7, thank you for your interest in using PySCF.
> 
> Yes, it is recommended to install WSL provided by the Windows App shop, and then install PySCF as you would do on a Linux system. I am actually also using PySCF on a Windows 11 system!
> 
> Hope this helps! Chong

Thank you for replying. Do I have to install all the python libraries and use the code files in linux sub system while the code editor runs in windows?
